src/main/webapp/app/core/about-us/about-us-routing.module.ts

```diff
@@ -0,0 +1,26 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { AboutUsComponent } from 'app/core/about-us/about-us.component';
+
+const routes: Routes = [
+    {
+        path: 'about',
+        children: [
+            {
+                path: '',
+                pathMatch: 'full',
```

Comment:
This looks a bit weird. Why aren't we defining component and data directly? Along the lines of just
```
path: 'about',
component: AboutUsComponent,
data: {
    authorities: [],
    pageTitle: 'aboutUs',
},
```

Reply:
Yes you are right, its weird. I started it with the idea I would need path children and forgot to adjust it later when it was not needed. Ill change it now